### PR TITLE
full review master

### PR DESCRIPTION
**Pull request by Rednoll**

### CODE REVIEW

File: tests/unit/services/web3/evm-web3.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import EvmWeb3Service from "@resolver/services/web3/evm-web3.service";
+import config from "@resolver/config";
+
+describe('evm-web3.service', () => {
+  test('SHOULD get web3 provider', () => {
+    expect(EvmWeb3Service.getWeb3(config.ETH_NODE)).not.toBe(undefined);
```
[the remark this answers is not in the document]
Comment (Ivan-dev-RED) [reply tests/unit/services/web3/evm-web3.service.spec.ts:6]: @Rednoll Didn't understand you

File: src/config.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export default {
+    REDEFINED_EMAIL_RESOLVER_CONTRACT_ADDRESS: "0xf20acFC20174acB8D08AAca523b30E8d54f64CfA",
+    ETH_NODE: "https://attentive-blue-choice.discover.quiknode.pro/a6d16bf50e42db77cd315b60e2c1f5c2386537e3/",
+    BSC_NODE: "https://bsc-dataseed.binance.org",
+    SOL_NODE: "mainnet-beta",
```
[the remark this answers is not in the document]
Comment (Ivan-dev-RED) [reply src/config.ts:5]: What for?

[the remark this answers is not in the document]
Comment (Rednoll) [reply src/config.ts:5]: For support configuration just via ENV without any code

File: tests/unit/services/web3/evm-web3.service.spec.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Rednoll) [reply tests/unit/services/web3/evm-web3.service.spec.ts:6]: I mean we should check following: getWeb3 return realy Web3 object. Not just not undefined

**Pull request closed without merging**